public/app/plugins/datasource/loki/querybuilder/components/LokiQueryEditorSelector.tsx

```diff
@@ -29,6 +30,13 @@ export const LokiQueryEditorSelector = React.memo<LokiQueryEditorProps>((props)
 
   const onEditorModeChange = useCallback(
     (newEditorMode: QueryEditorMode) => {
+      reportInteraction('user_grafana_loki_editor_mode_clicked', {
```

Comment:
```suggestion
      reportInteraction('grafana_loki_editor_mode_clicked', {
```

Reply:
I've added new name suggestion based on recommendations for naming from analytics team.